api_app/tests_ma/test_db/test_repositories/test_resource_repository.py

```diff
@@ -380,7 +380,7 @@ def test_validate_patch_with_good_fields_passes(template_repo, resource_repo):
 
     # check it's valid when updating a single updateable prop
     patch = ResourcePatch(isEnabled=True, properties={'vm_size': 'large'})
-    resource_repo.validate_patch(patch, template_repo, template)
+    resource_repo.validate_patch(patch, template_repo, template, "update")
```

Comment:
update, or upgrade?

Reply:
Good catch. The code only checks for install, so the test passed anyway.

 ```suggestion
    resource_repo.validate_patch(patch, template_repo, template, "upgrade")
```